pkg/optimizelyjson/optimizely_json.go

```diff
@@ -0,0 +1,98 @@
+/****************************************************************************
+ * Copyright 2020, Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+
+// Package optimizelyjson //
+package optimizelyjson
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+)
+
+// OptimizelyJSON holds the underlying structure of the object
+type OptimizelyJSON struct {
+	jsonRepr map[string]interface{}
```

Comment:
`jsonRepr` looks like holding the formatted value but I believe, it should be independent of any `Representation`, i mean the variable name should be something called `data` it's just a suggestion.